staging/src/k8s.io/apiserver/pkg/server/options/api_enablement.go

```diff
@@ -99,3 +98,9 @@ func unknownGroups(groups []string, registry *registered.APIRegistrationManager)
 	}
 	return unknownGroups
 }
+
+// Interface provides method to check whether given group is registered.
+type Interface interface {
+	// IsRegistered returns true if a group is registered.
+	IsRegistered(group string) bool
```

Comment:
This needs a better name. IsGroupRegistered?

Reply:
This is a child method of `func (m *APIRegistrationManager) IsRegistered(group string) bool {
` . I can not find a better way to pass interface if rename it.

Reply:
I the interface name is better, I am fine with this.

Reply:
done